src/Views/batchprocessor.html

```diff
@@ -104,18 +151,34 @@ <h3>Select Basin Characteristics</h3>
                         </tr>
                     </tbody>
                 </table>
+            </div><!-- End parameter selection -->
+
+        </div><!-- End submit batch -->
+        
 
-            </div>
-            <!-- End submit batch -->
+        <!-- Start batch status -->
+        <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
+            <p>This is the batch status tab</p>
+        </div> <!-- End batch status -->
+       
+</div> <!-- End of modal body -->
 
-            <!-- Start batch status -->
-            <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
-                <p>This is the batch status tab</p>
-            </div>
-            <!-- End batch status -->
+<!-- Start modal footer -->
+<div class="modal-footer">
+    <div class="row" style="justify-content: center;">
+        <!-- Button to submit batch -->
+        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch'" type="submit" class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch()"
+            ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment"><i

```

Comment:
To stay consistent, I would validate the .zip here

```suggestion
            ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment || this.submitBatchData.attachment.type != 'application/zip'"><i

```